src/survey/scss.coffee

```diff
@@ -0,0 +1,25 @@
+{go, map, tee, reject,
+include, Type, isType,
+Method,
+glob} = require "fairmont"
+
+{define, context, sass} = require "panda-9000"
```

Comment:
TODO: Change `sass` to `scss` when https://github.com/pandastrike/panda-9000/pull/16 is merged and deployed to NPM.